dts/arm/nordic/nrf52840.dtsi

```diff
@@ -52,6 +52,13 @@
 			status = "disabled";
 			label = "UART_1";
 		};
+
+		radio0: radio@40001000 {
+			compatible = "nordic,nrf52840-radio";
+			reg = <0x40001000 0x1000>;
+			interrupts = <1 1>;
+			iee802154_drv_label = "IEEE802154_nrf5";
```

Comment:
How to handle/describe multi-protocol abilities?

Reply:
I have Idea that It will be possible to ad here a `[technology]_drv_label` in the future like:
`ble_drv_label`, `ble_multiprotocol_drv_label` and so on. So far exists only this 15.4 driver - that why this is only label here.